GraphQL.Client.sln

```diff
@@ -68,6 +63,9 @@ EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.Client.Example", "examples\GraphQL.Client.Example\GraphQL.Client.Example.csproj", "{6B13B87D-1EF4-485F-BC5D-891E2F4DA6CD}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".github", ".github", "{98D4DDDD-DE15-4997-B888-9BC806C7416C}"
+	ProjectSection(SolutionItems) = preProject
+		.github\dependabot.yml = .github\dependabot.yml
```

Comment:
I notice the first commit in this PR is about adding dependabot configuration:
https://github.com/scopear/graphql-client/pull/7/commits/ad5bdb1093a041377c79e54cc693f09f0bc56fd0

1. Do we want to inherit their config (for daily runs) or run it at our own cadence?
2. I don't see .github/dependabot.yml added in this PR, when browsing the PR in github?  Was it gitignored somehow from getting pushed in your branch or something?
3. We still reference it here in the .sln file; will that generate errors if the yml file is missing?